src/rp2040.ts

```diff
@@ -880,6 +880,28 @@ export class RP2040 {
     }
   }
 
+  substractUpdateFlags(minuend: number, subtrahend: number) {
```

Comment:
Funny, I didn't know minuend/subtrahend was even a word! I learned something new :)

Can we make this method private?

Reply:
> I didn't know minuend/subtrahend was even a word!
the google's power :-)